docs/learn-identity.md

```diff
@@ -135,6 +135,22 @@ for the `max_fee` put the maximum you're willing to pay for these confirmations.
 If you don't know which registrar to pick, first check the available registrars by going to
 ["Chain State UI"]() and selecting `identity.registrars()` to get the full list.
 
+## Cancelling Judgements
+
+You may decide that you actually do not want to be judged by a Registar (for instance, because you
+realize you entered incorrect data or selected the wrong Registrar). In this case, after submitting
```

Comment:
```suggestion
You may decide that you actually do not want to be judged by a registrar (for instance, because you
realize you entered incorrect data or selected the wrong registrar). In this case, after submitting
```

don't think we need to capitalize registrar here